packages/@expo/log-box/src/logbox-rn-polyfill.tsx

```diff
@@ -0,0 +1,200 @@
+        componentCodeFrame: log.componentCodeFrame,
+        level: log.level,
+        type: log.type,
+        message: log.message,
+        stack: log.stack,
+        category: log.category,
+        componentStack: log.componentStack,
+        componentStackType: log.componentStackType,
+        codeFrame: log.codeFrame,
+        isComponentError: log.isComponentError,
+        extraData: log.extraData,
+        count: log.count,
+      };
+    });
+  }, [props.logs]);
+
+  const [open, setOpen] = React.useState(true);
+  const bundledLogBoxUrl = getBundledLogBoxURL();
+
+  const closeModal = (cb: () => void) => {
+    setOpen(false);
+    setTimeout(cb, Platform.select({ ios: 500, default: 0 }));
+  };
+
+  const onMinimize = () => closeModal(props.onMinimize);
+  const onDismiss = props.onDismiss;
+
+  const LogBoxWrapper = useMemo(
+    () =>
+      Platform.OS === 'ios'
+        ? ({ children }: { children: React.ReactNode }) => {
+            return (
+              <Modal
+                animationType="slide"
+                presentationStyle="pageSheet"
+                visible={open}
+                onRequestClose={onMinimize}>
+                {children}
+              </Modal>
+            );
+          }
+        : ({ children }: { children: React.ReactNode }) => <>{children}</>,
+    []
+  );
+
+  return (
+    <LogBoxWrapper>
+      <View
+        style={{
+          backgroundColor: Platform.select({ default: undefined, ios: Colors.background }),
+          pointerEvents: 'box-none',
+          top: 0,
+          flex: 1,
+        }}
+        collapsable={false}>
+        <LogBoxPolyfillDOM
+          platform={process.env.EXPO_OS}
+          devServerUrl={getBaseUrl()}
+          dom={{
+            useExpoDOMWebView: true,
+            sourceOverride: bundledLogBoxUrl ? { uri: bundledLogBoxUrl } : undefined,
+            contentInsetAdjustmentBehavior: 'never',
+            containerStyle: {
+              pointerEvents: 'box-none',
+              position: 'absolute',
+              top: 0,
+              left: 0,
+              right: 0,
+              bottom: 0,
+            },
+            style: {
+              flex: 1,
+            },
+            suppressMenuItems: ['underline', 'lookup', 'translate'],
+            bounces: true,
+            overScrollMode: 'never',
+          }}
+          fetchJsonAsync={async (input, init) => {
+            try {
+              const res = await fetch(input, init);
+              return await res.text();
+            } catch (e) {
+              throw e;
+            }
+          }}
+          reloadRuntime={() => {
+            // NOTE: For iOS only the reload is enough, but on Android the app gets stuck on an empty black screen
+            onMinimize();
+            setTimeout(() => {
+              DevSettings.reload();
+            }, 100);
+          }}
+          onCopyText={(text: string) => {
+            Clipboard.setString(text);
```

Comment:
```suggestion
            Clipboard.setString(text);
```
This is deprecated now, but probably fine for now.